src/components/AppComponent.native.tsx

```diff
@@ -21,7 +21,7 @@ import ModalAuth from './ModalAuth'
 import ErrorMessage from './ErrorMessage'
 import Content from './Content'
 
-import MultiGesture from './MultiGesture'
+import MultiGesture, { MultiGestureRef } from './MultiGesture.native'
```

Comment:
isnt this assumed to be native by the compiler?

Reply:
Yes, but the web multi-gesture component doesn't have a`MultiGestureRef` so we need to explicitly define the file in this case.